cockatrice/src/gamesmodel.cpp

```diff
@@ -197,6 +198,7 @@ QVariant GamesModel::headerData(int section, Qt::Orientation /*orientation*/, in
                 case Qt::TextAlignmentRole:
                     return Qt::AlignCenter;
             }
+            SWITCH_FALLTHROUGH;
```

Comment:
I'm sorry i didn't look at the code before bringing up the MOZ_FALLTHROUGH topic, but here a simple `break;` would have avoided the warning without the need of adding a new header file.

Reply:
If the other switch statement is the same I could just add a break there too. Remove the header file?

Reply:
yes, sorry again for the wasted time

Reply:
adding a `break` in these places makes the compiler think that the control could reach the end of the function `error: control reaches end of non-void function [-Werror=return-type]`

since the switch defaults to `return QVariant();` I think ill add it at the end of the function

Reply:
another idea would be adding default cases to both the switches returning QVariant(), this is done alot in the function above

Reply:
or, you can add `return QVariant()` in place of the breaks